fastlane/docs/Advanced.md

```diff
@@ -176,12 +176,45 @@ fastlane run notification message:"My Text" title:"The Title"
 
 To get the avaiable options for any action run `fastlane action [action_name]`. You might not be able to set some kind of parameters using this method.
 
-## Shell values
+## Shell interaction
+
+### Return value
+
 You can get value from shell commands:
 ```ruby
 output = sh("pod update")
 ```
 
+### Working directory
+
+'sh' action uses a different working directory than other Fastlane actions. As working directory, most Fastlane actions use the current directory as of the time when the fastlane script has been called. However, 'sh' action uses ./fastlane/ subdirectory of the current directory as its working directory. Furthermore, shell environment variable PWD does not reflect the working directory, while retrieved within 'sh' action. The behavior is illustrated by the following example:
+
+Action:
```

Comment:
I'd remove everything from here on, and just add a link to the issue. Make sure the link is an actual link the user can click on 👍